### PR TITLE
Fix nexmo app urls

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: temba/ivr/tests.py [hunk 1]
```diff
@@ -1100,34 +1112,91 @@ def test_incoming_call_nexmo(self, mock_create_application):
...
+        response = self.client.post(reverse('handlers.nexmo_call_handler', args=['answer', nexmo_uuid]),
+                                    json.dumps({}), content_type='application/json')
+        self.assertEqual(200, response.status_code)
+
+        response = self.client.post(reverse('handlers.nexmo_call_handler', args=['event', nexmo_uuid]),
+                                    '', content_type='application/json')
+        self.assertEqual(200, response.status_code)
+
+        response = self.client.post(reverse('handlers.nexmo_call_handler', args=['event', nexmo_uuid]),
+                                    json.dumps({}), content_type='application/json')
```
Comment (norkans7) [inline temba/ivr/tests.py]: this is to test the empty callbacks form Nexmo as they hit the URLs to make sure they exist
when creating the app

File: temba/ivr/clients.py [hunk 2]
```diff
@@ -57,7 +56,7 @@ def start_call(self, call, to, from_, status_callback):
             ChannelLog.log_ivr_interaction(call, "Started Nexmo call %s" % call.external_id, json.dumps(params),
                                            json.dumps(response), 'https://api.nexmo.com/v1/calls', 'POST')
 
-        except nexmo.Error as e:
+        except Exception as e:
```
Comment (ericnewcomer) [inline temba/ivr/clients.py]: Should use six.string_types(e) to stringify the exception instead of e.message.

File: temba/ivr/tests.py [hunk 3]
```diff
@@ -337,7 +337,7 @@ def test_ivr_recording_with_nexmo(self, mock_create_call, mock_create_applicatio
         for msg in messages:
             self.assertEquals(1, msg.steps.all().count(), msg="Message '%s' is not attached to exactly one step" % msg.text)
 
-        mock_create_call.side_effect = nexmo.Error('Error')
+        mock_create_call.side_effect = Exception('Error')
```
Comment (ericnewcomer) [inline temba/ivr/tests.py]: Should validate that this error message shows up in the channel log

File: temba/ivr/clients.py (hunk 2 above)
Comment (rowanseymour) [inline temba/ivr/clients.py]: `six.text_type(a)`

Comment (ericnewcomer) [inline temba/ivr/clients.py]: or that.

File: temba/ivr/tests.py (hunk 1 above)
Comment (ericnewcomer) [inline temba/ivr/tests.py]: gotcha.

**Comment by ericnewcomer** [comment]
Moving this PR to one that includes call hangups on expiration

**Pull request closed without merging**